site/app/controllers/admin/UsersController.php

```diff
@@ -244,4 +254,273 @@ public function updateRotatingSections() {
         $this->core->addSuccessMessage("Rotating sections setup");
         $this->core->redirect($return_url);
     }
+
+    private function getCsvOrXlsxData($filename, $tmp_name, $return_url) {
+        $content_type = FileUtils::getContentType($filename);
+        $mime_type = FileUtils::getMimeType($tmp_name);
+
+        if ($content_type === 'spreadsheet/xlsx' && $mime_type === 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet') {
+            $csv_file = "/tmp/".Utils::generateRandomString();
+            $xlsx_file = "/tmp/".Utils::generateRandomString();
```

Comment:
use [sys_get_temp_dir](http://php.net/manual/en/function.sys-get-temp-dir.php) instead of just assuming it's `/tmp`.